### DISCUSSION TITLE
HTML Input and form element does not seems to work in wysiwyg editor

### DISCUSSION THREAD

**Discussion by rajpatel2435** [flyntwp/flynt#569] [Q&A] [answered]
Hi @timohubois @aaronmeder 

Here I am facing the issue inside wysiwyg editor. In somehow the Form layout as well as input type does not seems to work in fronetend. could you please check this ?

Thanks

![image](https://github.com/user-attachments/assets/ba6aa542-23bc-4b86-b9fe-f6845de8158d)

![image](https://github.com/user-attachments/assets/dfa1928d-02be-4533-a10e-0d0165a7fb70)

**Comment by timohubois** [comment, marked as answer]
@rajpatel2435 it is may related to the escaping filter `|e('wp_kses_post')`: https://github.com/flyntwp/flynt/blob/ece2b904772dcdda76e6eb46adfca60c14dd80ad/Components/BlockWysiwyg/index.twig#L4

Have you tried to remove it?

More details about escaping can be found here: https://timber.github.io/docs/v2/guides/escaping/

**Comment by rajpatel2435** [comment]
Hi @timohubois 

It works Thanks for sharing that.

I have another question can we apply gzip compression on images on production build as it is working for svg not for png webp and other formats.

Thanks
